### ISSUE TITLE
Aduru-Darker

### ISSUE THREAD

**Issue by ejkarne** [godlyranchdressing/Aduru#1]
I know it's early days for this theme but I'm in love with it already. I've been using it on and off since the first commit and it's just so clean and attractive.

Now that I'm done gushing I'll get to the point. I need that Aduru-Darker. Right now I'm forced to choose between blinding myself or embracing the darkness. 
I propose a beautiful union that allows users to experience Aduru in all of it's glory without the eye socket melting brightness.

**Comment by godlyranchdressing** [comment]
Oh wow. Sorry for the late response. I didn't even know anyone was using it.

I'll work on it!

**Comment by ejkarne** [comment]
Well I know you from when I was checking out United and while I was looking at other themes I got to Minwaita which I had seen 1000 times before but I noticed for the first time, you're the one behind it! So I checked out your other repos and found Aduru and I absolutely love it. It takes everything wrong with Adwaita out of the equation but it still fits in perfectly. I actually have no criticisms of it.
